### PR TITLE
Artgalery and request fix

### PR DESCRIPTION
**Pull request by NadiaKishchuk**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/app/api/media/images.api.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import Image, { ImageCreate } from '@models/media/image.model';
 const ImagesApi = {
     getAll: () => Agent.get<Image[]>(`${API_ROUTES.IMAGES.GET_ALL}`),
 
-    getById: (id: number) => Agent.get<Image>(`${API_ROUTES.IMAGES.GET}/${id}`),
+    getById: (id: number) =>{/*  console.log(id);console.trace(); */  return Agent.get<Image>(`${API_ROUTES.IMAGES.GET}/${id}`);} ,
```
Comment (Tysyatsky) [inline src/app/api/media/images.api.ts:8]: rmv this comment pls

**Review by MaksBrat — APPROVED** [review]

File: src/app/api/media/images.api.ts (hunk 1 above)
Comment (NadiaKishchuk) [reply src/app/api/media/images.api.ts:8]: oh, thx)

File: src/features/StreetcodePage/ArtGalleryBlock/ArtGalleryBlock.styles.scss [hunk 2]
```diff
@@ -28,14 +28,40 @@
         width: pxToRem(1230px);
     }
 }
+@media screen and (max-width:1450px) {
+    .artGalleryWrapper {
+        .artGalleryContainer {
+            .artGalleryContentContainer{
```
Comment (Katerix) [inline src/features/StreetcodePage/ArtGalleryBlock/ArtGalleryBlock.styles.scss:34]: pls add a space before {

File: src/features/StreetcodePage/ArtGalleryBlock/ArtGalleryBlock.styles.scss [hunk 3]
```diff
@@ -28,14 +28,40 @@
...
+                .artGallerySliderContainer{
```
Comment (Katerix) [inline src/features/StreetcodePage/ArtGalleryBlock/ArtGalleryBlock.styles.scss:35]: same

File: src/features/StreetcodePage/ArtGalleryBlock/ArtGalleryBlock.styles.scss [hunk 4]
```diff
@@ -28,14 +28,40 @@
...
+                    width: pxToRem(830px);
+                }
+            }
+        }
+    }
+}
```
Comment (Katerix) [inline src/features/StreetcodePage/ArtGalleryBlock/ArtGalleryBlock.styles.scss:52]: same in this block

**Comment by Katerix** [comment]
Well done! Thank you for your work! please read my comments)

**Review by ormykhalyshyn — APPROVED** [review]

**Pull request merged**